src/components/Input/index.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { InputHTMLAttributes } from 'react';
+
+import styles from './styles.module.scss';
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+  inputRef?: React.Ref<HTMLInputElement>;
+  text?: string;
+  errorMessage?: string;
+}
+
+function InputText({ inputRef, text, errorMessage, name, ...rest }: Props) {
+  return (
+    <div className="column full-width">
+      <label className={`m-bottom-1 ${styles.label}`} htmlFor={name}>
+        {text}
+      </label>
+      <input className={styles.input} ref={inputRef} name={name} {...rest} />
+      <p className={styles.error}>{errorMessage || ''}</p>
```

Comment:
Are you sure you want to have this p tag always? You also can do: {errorMessage && <p className={styles.error}>{errorMessage}</p>}. But you can do it depending on the behavior you want

Reply:
Same for label. `{text && <label....>}`